src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java

```diff
@@ -71,7 +71,7 @@ public Item createGun(CommonProxy commonProxy) {
         ))
 		.withInformationProvider(stack -> Arrays.asList(
 		"Type: .50 Beowulf Rifle", 
-		"Damage: 7", 
+		"Damage: 10",
 		"Cartridge: .50 Beowulf",
```

Comment:
This seems like too much, that's a two shot kill and a one kill headshot (when headshots will be implemented or if they are already)

Reply:
Weapons of the same type deal 10 damage

Reply:
I have a question that once occurred to me, when players wore diamond suits, it was difficult for weapons to cause damage to them, I'm trying to fix it

Reply:
> I have a question that once occurred to me, when players wore diamond suits, it was difficult for weapons to cause damage to them, I am planning to try to solve it

If you do please create another branch and do it as a separate PR

Reply:
\Weaponlib  ServerEventHandler. java: 246
I understand that the problem is caused here